Tools/build.bat

```diff
@@ -76,8 +76,8 @@ for %%N in (%SUPPORTED_BUILD_SPECS%) do (
   )
 )
 
-rem Silently exit if the build target could not be matched
-goto :eof
+rem Return error if the build target could not be matched
+goto :fail
```

Comment:
Hi Ladi, thanks for review!

It was not clear for me what was that code for. With your explanations this makes sense. Would omitting this chunk fully resolve the issue?